addons/web/static/src/core/commands/command_palette_dialog.js

```diff
@@ -29,6 +29,6 @@ export class CommandPaletteDialog extends Dialog {
 CommandPaletteDialog.renderHeader = false;
 CommandPaletteDialog.renderFooter = false;
 CommandPaletteDialog.size = "modal-md";
-CommandPaletteDialog.contentClass = "o_command_palette";
+CommandPaletteDialog.contentClass = "o_command_palette border-0 overflow-hidden";
```

Comment:
Sure about the need for those 2 classes ?
- purpose of the `overflow-hidden`?
- are the default border really a problem? (`1px solid rgba(0, 0, 0, 0.2)`)